mappings/com/mojang/blaze3d/MemoryUtil.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_ilpgzdyq net/minecraft/client/util/GlAllocationUtils
+CLASS net/minecraft/unmapped/C_ilpgzdyq com/mojang/blaze3d/MemoryUtil
```

Comment:
This should probably be called AllocationUtils. The methods inside are not specific to OpenGL, and MemoryUtil conflicts with a commonly used LWJGL class.

Reply:
`NativeByteBufferUtils`?